src/r3f/TilesRenderer.jsx

```diff
@@ -165,6 +171,12 @@ export const TilesRenderer = forwardRef( ( props, ref ) => {
 	// add the camera
 	useEffect( () => {
 
+		if ( tiles === null ) {
+
+			return;
+
+		}
+		
```

Comment:
This looks redundant with the `tiles === null` condition below

Reply:
sorry about that